pyproject.toml

```diff
@@ -15,14 +15,14 @@ keywords = ["volttron", "agent", "messagebus", "application"]
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
 requests = "^2.26.0"
-volttron-client = "^0.3.8"
+#volttron-client = "^0.3.8"
 
 [tool.poetry.dev-dependencies]
 pre-commit = "^2.13.0"
 pytest = "^6.2.4"
 black = "^21.6b0"
 flake8 = "^3.9.2"
-#volttron-client = {path="/repos/volttron-client", develop=true}
+volttron = {path="../volttron-core", develop=true}
```

Comment:
This should actually be a volttron version...I think

Reply:
Once we push a volttron package to pypi we can change this but I wanted to have at least one agent code that everyone can install and test with the volttron-core repo